### DISCUSSION TITLE
Multiple account switching

### DISCUSSION THREAD

**Discussion by RickCogley** [outline/outline#1872] [Self Hosting] [answered]
Since I can only turn off publishing to public globally, and I don't want to allow it for all members of my org, I am considering getting two hosted accounts. One would have that turned off, and would be for everyone, and the other account would have that turned on, but only be for a couple of people. We would use the latter for things like public-facing FAQs and so on. 

If I were to do this, is there an easy way in the app to switch between the two accounts? I'd probably want to use my same work email for both. 

Please let me know, thanks. :)

**Comment by tommoor** [comment, marked as answer]
Hey Rick, we have an open roadmap item to add disabling of public sharing to individual collections – would this cover your usecase? I'd prefer that we can keep your team together rather than having two accounts :)

**Reply by RickCogley** [reply]
Yes, I believe it would!

**Reply by tommoor** [reply]
I'm confident we can get that deployed this week, stay tuned

**Reply by tommoor** [reply]
This feature has now been deployed – you can disable sharing for any individual collection in the "Edit…" screen

![image](https://user-images.githubusercontent.com/380914/107727946-7b5a9300-6ca1-11eb-87c2-9f112bdbc717.png)
